webhook/client.go

```diff
@@ -27,10 +27,10 @@ var (
 	ErrNoValidSignature error = errors.New("Webhook had no valid signature")
 )
 
-// Computes a webhook signature using Stripe's v1 signing method. See
+// ComputeSignature computes a webhook signature using Stripe's v1 signing method. See
 // https://stripe.com/docs/webhooks#signatures
-func computeSignature(t time.Time, payload []byte, secret string) []byte {
-	mac := hmac.New(sha256.New, []byte(secret))
+func ComputeSignature(t time.Time, payload, secret []byte) []byte {
```

Comment:
Thanks a lot for trying to follow the advice I was giving in #497!

Contrary to what I said there though, I now see that unfortunately changing the signature to `[]byte` ends up breaking compatibility on a lot of these already-exported functions. It's also still not perfectly consistent because `header` in some of the functions below is still a string.

Sorry for the reversal, but for those reasons, I think it actually might be better to leave `secret` here as a `string`, at least for the time being. Would you mind changing it back?